docs/conf.py

```diff
@@ -13,7 +13,7 @@
 # General information about the project.                         
 project = 'emloop-tensorflow' 
 copyright = '2018, Iterait a.s.'                     
-author = 'Blazek Adam, Belohlavek Petr, Matzner Filip'
+author = 'Blazek Adam, Belohlavek Petr, Matzner Filip, Bedrich Pisl'
```

Comment:
nice :)

Reply:
Nice except that Beda's name and surname are in the wrong order. :-)

Reply:
whatever